### ISSUE TITLE
Accessibility - Adding ARIA Roles to enable basic navigation

### ISSUE THREAD

**Issue by mmmavis** [mozilla/lightbeam#343]
Adding ARIA Roles to enable basic navigation within Collusion Add-on.  ( from #232 )

For screen reader users, they should
- [x] Be able to go to List View
- [x] Navigate(read) through the table on List View

We can add more support once we have completed the above items.

**Comment by fmarier** [comment]
This looks like it's done.

If further accessibility work should be done, please file a follow-up bug.
